tests/system_tests/appstart_systemtests.py

```diff
@@ -42,6 +41,12 @@ def setUpClass(cls):
 
         This depends on a properly set up docker environment.
         """
+        
+        SystemTests.base_image="appstart_systemtest_devappserver"
```

Comment:
Also, this should probably be a constant, and you shouldn't need to qualify it with the class name (frankly, I'm surprised that you _can_ qualify it with the classname as this shouldn't be defined at this point):
   BASE_IMAGE = 'appstart...'